### DISCUSSION TITLE
An issue I have with the song select menu

### DISCUSSION THREAD

**Discussion by thybomb** [ppy/osu#28956] [Ideas] [answered]
In the song select menu, I usually have maps sorted by difficulty, and because of this, often times a lot of maps will show up with just one difficulty instead of the whole mapset, and I noticed that even when there is only one difficulty of a mapset, I cannot click on the box with the title/author of the song to start it (aka the box I'm hovering over in the ss), only the box with the difficulty name and mapper. Can this be changed so that when there is only one difficulty available, the box with the title and author is clickable to start the map?

![Screenshot 2024-07-19 161559](https://github.com/user-attachments/assets/be365869-2de8-4aa7-8917-9f1ff2e71da5)

**Comment by Joehuu** [comment, marked as answer]
Will be resolved in the song select redesign. You can see progress in https://github.com/ppy/osu/issues/22204.
